x/upgrade/test/integration_test.go

```diff
@@ -54,12 +53,8 @@ func (s *UpgradeTestSuite) SetupSuite() {
 		accounts[i] = tmrand.Str(9)
 	}
 
-	tmCfg := testnode.DefaultTendermintConfig()
-	tmCfg.Consensus.TimeoutCommit = 3 * time.Second
```

Comment:
we might need to add some more checks to ensure this is doing what its supposed to. The reasoning was the same as the square size test, where we changed the voting period to be 2s. We want to have a block time larger than that so that we can vote.

that's what the 
```go
WithGenesisOptions(testnode.ImmediateProposals(s.ecfg.Codec))
```
does at least.

 Since the upgrade module here removes the ability to upgrade, there should not be a way to do get a proposal through

Reply:
Sorry I'm lost.

> We don't have to have a block time longer than that, however it is convenient.

Why does the test need a block time longer than voting period? FWIW voting period is 5 seconds per [these lines](https://github.com/rootulp/celestia-app/blob/16fbdfe2d17832a282b46b2297370c2eab3715fa/test/util/testnode/genesis_options.go#L38-L39).

> Since the upgrade module here removes the ability to upgrade, there should not be a way to do get a proposal through

Agreed. I assumed that's what the tests in this file are doing and if they pass without overriding `TimeoutCommit` then I don't see why it's necessary.

Reply:
I'm going to revert the changes here 🤷‍♂️ idk why TimeoutCommit is overridden but happy to include that in a code comment

Reply:
Reverted in 038a5d8af39746bb3a150cafc04e9060e1f0e5f5